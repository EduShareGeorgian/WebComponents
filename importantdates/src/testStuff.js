import React from 'react';


class TestStuff extends React.Component {

  render() {
    return (
        <div id="HeaderContainer" class="ms-Grid ms-bgColor-white WebPartOuterContainer">
            <div class="ms-Grid-row">
                <div class="ms-Grid-col ms-u-sm12 WebPartHeading" style="background-image: url('../Images/calendar.svg')">
                    <h2>Important Dates</h2>
                </div>
            </div>             
        </div>                      
    )
  }
}

export {TestStuff}