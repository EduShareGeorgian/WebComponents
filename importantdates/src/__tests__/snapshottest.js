import {TestStuff} from '../testStuff';
import React from "react"
import renderer from "react-test-renderer"

describe('application launch', () => {
  it('should render App', () => {
    const tree = renderer.create(
          <TestStuff></TestStuff>
      ).toJSON();
      expect(tree).toMatchSnapshot("ImportantDates");
  });
});
