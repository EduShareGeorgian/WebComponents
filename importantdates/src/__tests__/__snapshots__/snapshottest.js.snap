// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImportantDates 1`] = `
<div
  class="ms-Grid ms-bgColor-white WebPartOuterContainer"
  id="HeaderContainer"
>
  <div
    class="ms-Grid-row"
  >
    <div
      class="ms-Grid-col ms-u-sm12 WebPartHeading"
      style="background-image: url('../Images/calendar.svg')"
    >
      <h2>
        Important Dates
      </h2>
    </div>
  </div>
</div>
`;
